unified-runtime/source/adapters/hip/usm.cpp

```diff
@@ -383,12 +383,12 @@ bool ur_usm_pool_handle_t_::hasUMFPool(umf_memory_pool_t *umf_pool) {
 
 UR_APIEXPORT ur_result_t UR_APICALL urUSMPoolCreate(
     /// [in] handle of the context object
-    ur_context_handle_t Context,
+    ur_context_handle_t /*Context*/,
     /// [in] pointer to USM pool descriptor. Can be chained with
     /// ::ur_usm_pool_limits_desc_t
-    ur_usm_pool_desc_t *PoolDesc,
+    ur_usm_pool_desc_t * /*PoolDesc*/,
     /// [out] pointer to USM memory pool
-    ur_usm_pool_handle_t *Pool) {
+    ur_usm_pool_handle_t * /*Pool*/) {
```

Comment:
Looking at this again I think this would fail if `UMF_ENABLE_POOL_TRACKING` is set, we should use `[[maybe_unused]]` for these too.

Reply:
Gone. Thanks